cmd/common.go

```diff
@@ -152,102 +150,40 @@ type AndroidManifest struct {
 	Package string `xml:"package,attr"`
 }
 
-func addBuildConstantSourceFile() {
-	currentEmbedderTag, err := versioncheck.CurrentGoFlutterTag(buildPath)
-	if err != nil {
-		fmt.Printf("hover: %v\n", err)
-		os.Exit(1)
-	}
-
-	semver, err := version.NewSemver(currentEmbedderTag)
-	if err != nil {
-		fmt.Printf("hover: faild to parse 'go-flutter' semver: %v\n", err)
-		os.Exit(1)
-	}
-
-	minimumGoFlutterRelease, _ := version.NewSemver("0.30.0")
-	if semver.LessThan(minimumGoFlutterRelease) {
-		return
-	}
-
-	buildVersionPath := filepath.Join(buildPath, "cmd", "version.go")
-	if _, err := os.Stat(buildVersionPath); err == nil {
-		// file exist
-		err := os.Remove(buildVersionPath)
-		if err != nil {
-			fmt.Printf("hover: Failed to delete %s: %v\n", buildVersionPath, err)
-			os.Exit(1)
-		}
-	}
-
-	buildVersionFile, err := os.Create(buildVersionPath)
-	if err != nil {
-		fmt.Printf("hover: Failed to create %s: %v\n", buildVersionPath, err)
-		os.Exit(1)
-	}
-
+// androidOrganizationName fetch the android package name (default:
+// 'com.example').
+// Can by set upon flutter create (--org flag)
+//
+// If errors occurs when reading the android package name, the string value
+// will correspond the default flutter generated value ('com.example').
+func androidOrganizationName() string {
 	// Default value
 	androidManifestFile := "android/app/src/main/AndroidManifest.xml"
 
 	// Open AndroidManifest file
 	xmlFile, err := os.Open(androidManifestFile)
 	if err != nil {
 		fmt.Printf("hover: Failed to retrieve the organization name: %v\n", err)
-		writeVersionFle(currentEmbedderTag, "com.example", buildVersionFile)
-		return
+		return "com.example"
```

Comment:
I'd feel more comfortable if  we return a value that indicates an issue.
If a user doesn't read the hover output, and in the app gets "com.example", they'll likely ctrl+f for "com.example" in their codebase to find where they 'forgot' to change the package name.... It's confusing.
Maybe return "unknown" here? Or even more specific "hover.failed.to.retrieve.package.name"? :sweat_smile:

Reply:
Yea, I see your concern.
I'll go with `build.hover.example`! :smiley: